glide.yaml

```diff
@@ -11,7 +11,10 @@ import:
 - package: k8s.io/kube-aggregator
   version: kubernetes-1.12.5
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.12.5
+  repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
+  version: origin-4.0-kubernetes-1.12.4 # for nullable PR, instead of kubernetes-1.12.5
```

Comment:
I thought the change that allowed generations to be `null` was in library-go? I think I heard Michal Fojtik once say that we should avoid depending on our own kube forks. Could this be solved by kube 1.13 rebase?

Reply:
checking on it.. will report back

Reply:
but many other operators are depending on that fork atm :woman_shrugging:

Reply:
> I think I heard Michal Fojtik once say that we should avoid depending on our own kube forks.

this is true until we have backports we depend on :-/

Reply:
the nullable change is actually in kubernetes-apiextensions-apiserver, this fork is okay for now because we don't yet have the functionality in 1.13